public_html/json_schema_build.php

```diff
@@ -304,7 +305,45 @@
             </div>
         </div>
     </div>
-
+    <!-- Moving multiple params into group modal -->
+    <div class="modal fade" id="multi_select_modal" tabindex="-1" role="dialog" aria-hidden="true">
+        <div class="modal-dialog modal-dialog-scrollable modal-lg" role="document">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <h4>Move parameters into <span></span></h4>
+                    <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                      <span aria-hidden="true">&times;</span>
+                    </button>
+                </div>
+                <div class="modal-body">
+                    <table class="table table-sm">
+                        <thead>
+                            <tr>
+                            <th scope="col" style="width:1%"></th>
+                            <th scope="col">Parameter</th>
+                            <th scope="col">Description</th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                        </tbody>
+                    </table>
+                    
+                </div>
+                <div class="modal-footer">
+                    <div class="btn-group" role="group">
+                        <button type="button" class="btn btn-light" id="select_all_params">Select all</button>
+                        <button type="button" class="btn btn-light" id="deselect_all_params">Deselect all</button>
+                    </div>
+                    <div class="col text-right pr-0">
+                    <button type="button" class="btn btn-secondary" data-dismiss="modal">Cancel</button>
+                    <button type="button" class="btn btn-primary disabled" id="move_params">
+                        Move into <span></span>
```

Comment:
My intention with this new `<span>` was to have the group name in the button, where you actually click. This makes the intention a bit more explicit.

However, thinking about it now the group names can be quite long so I disagree with my past self. So should probably revert this..

```suggestion
                        Move parameters
```